src/pybind/mgr/dashboard/controllers/cephfs.py

```diff
@@ -469,6 +468,30 @@ def rm_snapshot(self, fs_id, path, name):
         cfs = self._cephfs_instance(fs_id)
         cfs.rm_snapshot(path, name)
 
+    @RESTController.Resource('GET')
+    def get_quotas(self, fs_id, path):
```

Comment:
Strange `get_quota` exists too in line 433 and it looks completely similar except for the name. (Similar case with `set_quotas`.)

Could you please remove the change that's not needed?

Reply:
Yeah, I can remove it; it was from a PR that was merged recently and came in on the most recent rebase.